### DISCUSSION TITLE
Hello, I want to ask why I used the Spectrogram plug-in 2. My spectrum colleague looks like this

### DISCUSSION THREAD

**Discussion by Colorful-Toujours** [katspaugh/wavesurfer.js#2804] [Q&A] [answered]
![image](https://github.com/katspaugh/wavesurfer.js/assets/70690014/043d3c48-46e0-465a-9796-09858794b7a7)

**Comment by katspaugh** [comment]
You need to either increase the decoded sample rate (set a higher `sampleRate` wavesurfer option), or set a lower `frequencyMax` option of the spectrogram plugin.

**Reply by Colorful-Toujours** [reply]
![image](https://github.com/katspaugh/wavesurfer.js/assets/70690014/ee2afb6f-beb6-459a-8395-6470be2fe2d4)
I set wavesufer option sampleRate to 20000 has no effect, setting frequencyMax to 5 looks like this

**Reply by katspaugh** [reply, marked as answer]
It should be 5kHz, not 5 Hz. `frequencyMax : 5000`

**Reply by Colorful-Toujours** [reply]
Thank you so much for your guidance it's great!

**Reply by Colorful-Toujours** [reply]
Hello，wavesufer.js Spectrogram plugin If the audio is dual-channel, can the Spectrogram plugin be divided into two channels and rendered separately as shown in the figure
![image](https://github.com/katspaugh/wavesurfer.js/assets/70690014/6631e78e-f4aa-4232-88f6-46e2611afb25)
Now it looks like channel =1 in the spectrogram plugin
